tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -229,9 +235,122 @@ setup() {
     assert_rootfs_count "$node" "$sandbox_id" "1"
 }
 
+function setup_image_pull_secret_for_nydus_snapshotter_metadata_pull() {
+    kubectl delete secret cococred --ignore-not-found
+    kubectl create secret docker-registry cococred --docker-server=$(echo "$AUTHENTICATED_IMAGE" | cut -d':' -f1) \
+    --docker-username="${AUTHENTICATED_IMAGE_USER}" --docker-password="${AUTHENTICATED_IMAGE_PASSWORD}"
+}
+
+function setup_kbs_credentials() {
+    image=$1
+    user=$2
+    password=$3
+
+    if [ "${KBS}" = "false" ]; then
```

Comment:
@stevenhorsman I realized the skips within this function doesn't matter if we don't skip the `teardown()`. I've some ideas on how to fix it but let's worry about it now. Priority is making the authentication work :)